benchmarks/README.md

```diff
@@ -1,6 +1,56 @@
-ReadabiliPy - TuringBench
+Benchmarking [ReadabiliPy](https://github.com/alan-turing-institute/ReadabiliPy/tree/master) with containers
```

Comment:
Doesn't this refer to itself? Maybe that's still useful though.

Reply:
Yes, this README gets shown on DockerHub